scratchapi2/project.py

```diff
@@ -1,3 +0,0 @@
-""" This is no longer supported. Use user.Project() instead. """

```

Comment:
Deleted the file completely. It hasn't been released - no need to worry about backwards compatibility yet.